src/mscorlib/src/System/Collections/Generic/KeyValuePair.cs

```diff
@@ -62,5 +62,12 @@ public struct KeyValuePair<TKey, TValue> {
             s.Append(']');
             return StringBuilderCache.GetStringAndRelease(s);
         }
+
+        //[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
We should move EditorBrowsableAttribute to CoreLib. It is in CoreLib on CoreRT already, so it is a good idea to get the two in sync.